src/RiotPls.DataDragon/Converters/DoubleArrayJsonConverter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace RiotPls.DataDragon.Converters
+{
+    public sealed class DoubleArrayJsonConverter : JsonConverter<double[]>
```

Comment:
We proly want to support T[] instead of just double[] in case there are more cases like this.